lib/events/dynamoevents/dynamoevents.go

```diff
@@ -732,6 +733,20 @@ func (notReadyYetError) Error() string {
 	return "The DynamoDB event backend is not ready to accept queries yet. Please retry in a couple of seconds."
 }
 
+func eventFilterList(amount int) string {
+	list := "("
+
+	if amount != 0 {
+		list += ":eventType0"
+	}
+
+	for i := 1; i < amount; i++ {
+		list += fmt.Sprintf(", :eventType%d", i)
+	}
+
+	return list + ")"
+}
```

Comment:
This function needs comments please on what it's doing and what kind of string it is constructing - without the context it's very unclear currently. I would expand on the following:
* What does it return? Seems like some filter expression but it's unclear.
* What is `amount`?
* What are `eventType0` and `eventTypeX` and why we add one or another based on "amount".

Reply:
Also, if you wrote it this way (and if I understood correctly what it's doing), I think it would be a bit clearer to the reader:

```go
func eventFilterList(amount int) string {
  var eventTypes []string
  for i := 0; i < amount; i++ {
    eventTypes = append(eventTypes, fmt.Sprintf(":eventType%d", i))
  }
  return "(" + strings.Join(eventTypes, ", ") + ")"
}
```

Would this be equivalent?

Reply:
Yep, I've gone ahead and rewritten the function and updated the doc. Please have another look.